server/ai_live_video.go

```diff
@@ -170,3 +172,54 @@ func startControlPublish(control *url.URL, params aiRequestParams) {
 		}
 	}()
 }
+
+func startEventsSubscribe(ctx context.Context, url *url.URL, params aiRequestParams) {
+	subscriber := trickle.NewTrickleSubscriber(url.String())
+
+	clog.Infof(ctx, "Starting event subscription for URL: %s", url.String())
+
+	go func() {
+		for {
+			clog.Infof(ctx, "Attempting to read from event subscription for URL: %s", url.String())
+			segment, err := subscriber.Read()
+			if err != nil {
+				clog.Infof(ctx, "Error reading events subscription: %s", err)
+				monitor.DeletePipelineStatus(params.liveParams.stream)
+				return
+			}
+
+			clog.Infof(ctx, "Successfully read segment from event subscription for URL: %s", url.String())
+
+			body, err := io.ReadAll(segment.Body)
+			segment.Body.Close()
+
+			if err != nil {
+				clog.Infof(ctx, "Error reading events subscription body: %s", err)
+				continue
+			}
+
+			stream := params.liveParams.stream
+
```

Comment:
WDYT of keeping that log of every event received? Could be useful for debugging through loki if there's any issues with kafka. We send 1 event every 10s per stream which seems a reasonable log amount.

I'd also add the log exactly here, before any of the checks below that might skip processing of the event. Add the `stream` as a log field, if it's empty it's empty in the log.